internal/http/services/owncloud/ocdav/copy.go

```diff
@@ -52,18 +52,35 @@ func (s *svc) handlePathCopy(w http.ResponseWriter, r *http.Request, ns string)
 	ctx, span := rtrace.Provider.Tracer("reva").Start(r.Context(), "copy")
 	defer span.End()
 
+	// If is a third party copy
+	if r.Header.Get("ThirdPartyCopy") == "T" {
```

Comment:
Is this header documented somewhere? I don't seem to find it in GFAL2 nor in Davix...

Reply:
Nope. It is not. I have used this in davix for reva specific queries to differentiate between a remote and a local transfer (site managed by same server). So I used this additional header to differentiate.

Reply:
I see. But then as we can't assume clients use Davix, we should stick to probing the specific HTTP TPC headers: I'd check whether the `TransferAuthorization` or `TransferX-Access-Token` headers are present in order to assert this is a TPC.

Reply:
Also a more general comment, which goes beyond GSoC: we will have to see how to use `TransferAuthorization` / `Authorization` as headers, because when we interact with grid sites they will likely send us (Reva) an `Authorization` header, not an `X-Access-Token` one, as their implementation again does not necessarily use Davix and would just stick to the HTTP TPC specs (exactly like we use the go http client in Reva, not Davix).

Reply:
1. Sure, I will change to using `TransferHeaderAuthorisation` or `TransferHeaderX-Access-Token` to attest it as TPC.
2. Yeah, I had this in my head. We will figure it out based on the authentication mechanism we finalise.

Reply:
@dynamic-entropy can you remove the reference to `ThirdPartyCopy` and replace it with `TransferHeaderAuthorization`?

Reply:
Well, yeah, since the other copy flow does not use/mention this header, `TransferHeaderAuthorization` will do the work.